ts/private/ts_lib.bzl

```diff
@@ -164,15 +164,11 @@ def _join(*elements):
         return "/".join(segments)
     return "."
 
-def _strip_external(path):
-    return path[len("external/"):] if path.startswith("external/") else path
-
 def _relative_to_package(path, ctx):
-    for prefix in [ctx.bin_dir.path, ctx.label.workspace_name, ctx.label.package]:
-        prefix += "/"
-        path = _strip_external(path)
-        if path.startswith(prefix):
-            path = path[len(prefix):]
+    path = path.removeprefix("external/").removeprefix(ctx.bin_dir.path + "/")
```

Comment:
I think we should only remove "external/" once now like you suggest. It seems like it was just wrong before

Reply:
I think it would only be needed once *after* removing `ctx.bin_dir.path`?

Reply:
Although we have no tests breaking if I remove it so I think I'd prefer leaving it for now and just putting a TODO?

Reply:
Unless you're more confident then I am? 😅 

For now I just put a TODO...